lib/galaxy/tool_util/linters/tests.py

```diff
@@ -13,7 +13,7 @@ def lint_tsts(tool_xml, lint_ctx):
         lint_ctx.info("No tests found, that should be OK for data_sources.")
 
     num_valid_tests = 0
-    for test in tests:
+    for test_idx, test in enumerate(tests):
```

Comment:
```suggestion
    for test_idx, test in enumerate(tests, start=1):
```

and drop all the following ` + 1` ?